### PR TITLE
Theta Project Update v1.1 for New Game Mode

### PR DESCRIPTION
**Pull request by lngredients**
- Changed Theta Project Armor to a cooler looking armor set.
- Theta armor should be capable of holding most weapons in its suit slot.
- Gave the theta project closet a bullpup rifle and three mags to equip their creation, or be killed by it.
- The armor gives a slight boost to walk speed.
- Theta Undersuit was replaced and now will actually work.
- Slightly Changed Theta Project equipment names so they arn't so long.
- Deleted Theta Boots since they are redundant
- Decreased armor in Theta Arm Guards

### CODE REVIEW

**Comment by lngredients** [comment]
Don't merge this just yet.

**Comment by lngredients** [comment]
If the CI passes this is now ok to merge.

**Comment by lngredients** [comment]
How do I add a change log?

**Comment by bloxgate** [comment]
Take a look at  html/changelogs/__CHANGELOG_README.txt 

It's really simple.

**Comment by lngredients** [comment]
Let me know if that is good Blox.

**Comment by lngredients** [comment]
I would love for this to be reviewed if possible.

**Comment by caelaislinn** [comment]
Map changes are a little more difficult to review. I'll have a look now

**Comment by lngredients** [comment]
Thanks Cael.

**Comment by lngredients** [comment]
Line 642 should contain all the relevant changes from this PR I believe.

**Comment by caelaislinn** [comment]
Maps aren't reviewed by looking at the text. We download the map and go through it in the editor

**Comment by lngredients** [comment]
Fair enough, I noticed some typos and other things so I am pushing a little update.

**Review by caelaislinn — DISMISSED** [review]
I like what you've done here, these seem to be some good balance changes. They need to be done in the code though, not on the map. The changelog has too many entries for the limited number of changes as well.

**Comment by lngredients** [comment]
Well I'm not quite well versed enough to create entirely new objects through the code. I used variable changes because I'm more familiar with how they work, and it takes less time then parsing code.

**Comment by lngredients** [comment]
Is there a downside to doing them on the map?

**Comment by caelaislinn** [comment]
It's hard for maintainers to review, and its hard for anyone to make further balance tweaks later.

**Comment by lngredients** [comment]
Blurgh, well then how would I go about making these items in the code?

**Comment by caelaislinn** [comment]
All of the Theta gear appears to be pre-existing items except for 2/4 of the organs. Create a new subtype of the pre-existing weapons and armour in a new file somewhere inside code/modules/halo/, then change your variables to match the values you want set.

File: html/changelogs/Ingredients-Theta Project Upgrades.yml [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+  - tweak: "Theta Undersuit was replaced and now will actually work."
+  - tweak: "Slightly Changed Theta Project equipment names so they arn't so long."
+  - tweak: "Decreased armor in Theta Arm Guards."
+  - tweak: "Decreased armor from Theta Boots to be more in line with the Arm Guards."
+  - tweak: "Made armor thicker on Theta armor to facilitate longer usage time. And to offset no shields."
+  - tweak: "5 armor thickness to boots and gloves."
+  - tweak: "10 armor thickness on Theta Undersuit."
+  - tweak: "Minimum breath pressure halved for augmented lungs."
+  - tweak: "Slightly increased lung max damage because it was formally at the native damage levels"
+  - tweak: "Added descriptions to all the augmented organs."
```
Comment (DoctorOctopusTheOctopi) [inline html/changelogs/Ingredients-Theta Project Upgrades.yml]: Should be shrunk down. You can smash all things that relate together, "decreased armor in Theta Arm Guards." can smash the "Decreased armor from theta boots" in with the theta arm guards.. For example
_"Decreased armor in Theta Arm Guards and boots."_  
You can remove the 5 armor thickness and 10 armor thickness lines from the changelog, people will get confused.

**Comment by lngredients** [comment]
Apparently I don't have the newest version of Python? Is that what is going on?

**Comment by lngredients** [comment]
I don't understand why the CI is no longer passing.

**Comment by lngredients** [comment]
Once this last CI is finished this PR should be good to go.

**Comment by lngredients** [comment]
It is done! I tested it and the new gun icon seems to be working even while wielded. All the armor is well in place and the organs are on point, though I can't really put them inside myself they are functionally the same as before. Someone for the love of all that is Halo review my PR! *sobs*

**Review by BDpuffy420 — APPROVED** [review]
lgtm

**Review by caelaislinn — APPROVED** [review]
Looks fine. Still think there's far too many changelog entries. You don't need 13 entries for balance tweaks.

**Pull request merged**